the-blue-alliance-ios/View Elements/Events/RankingTableViewCell.xib

```diff
@@ -25,40 +25,52 @@
                                 <rect key="frame" x="0.0" y="0.0" width="68" height="100.5"/>
```

Comment:
Can we set the root stack view (the horizontal one) to have an alignment of top? Maybe add like 4px spacing to the other two stack views so the text gets some room to breathe. But that might help some of the spacing weirdness